.gitignore

```diff
@@ -1,6 +1,7 @@
 /_build
 /deps
 /doc
+/lib/generated
```

Comment:
During development I've been running the mix task a lot against different inputs and looking at the generated code. It was a bit distracting having to selectively `git add` certain things. I don't think this will affect users of the library.